x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.test.ts

```diff
@@ -436,6 +436,7 @@ describe('Log threshold executor', () => {
               group: null,
               matchingDocuments: 10,
               isRatio: false,
+              reason: '10 log entries in the last 5 mins. Alert when (more than 5).',
```

Comment:
@fkanout Let's use here the new reason format `Alert when > 5.`

Reply:
@mgiota these are generated messages, not written manually. And the values are coming from `ComparatorRT` directly and not from the i18n function. 

https://github.com/fkanout/kibana/blob/31946d5a477a307493d58ddf1fdfa837e5a8253b/x-pack/plugins/infra/common/alerting/logs/log_threshold/types.ts#L22

Reply:
I think these tests need to be refactored to use the i18n. Unless it's intended to make the test langue agnostic. And my new comparator symbol function is not appearing here yet (I'm not sure why).

Reply:
I will @elasticmachine merge upstream to see if I can map the test to my new function